### PR TITLE
Fix:Missing key prop task-6 issue(#132)

### PR DESCRIPTION
**Pull request by CulturalProfessor**
## Fixed Missing key prop in task-6 issue(#132)

Screenshot from 2023-02-16 19-33-51

### CODE REVIEW

File: src/components/Attachments/Attachment.js [hunk 1]
```diff
@@ -24,9 +24,7 @@ const Attachment = ({ attachment, host }) => {
       <p>{attachment?.description}</p>
       <p>
         <Icon name="file" size="x20" />
-        <a href={`http://localhost:3000${attachment.title_link}`}>
-          {attachment.title}
-        </a>
+        <a href={`${host}${attachment.title_link}`}>{attachment.title}</a>
```
Comment (sidmohanty11) [inline src/components/Attachments/Attachment.js:27]: I think you've already added this change?

**Review by sidmohanty11 — CHANGES_REQUESTED** [review]
Thanks @CulturalProfessor for the PR

File: src/components/Attachments/Attachment.js (hunk 1 above)
Comment (CulturalProfessor) [reply src/components/Attachments/Attachment.js:27]: Yes

Comment (sidmohanty11) [reply src/components/Attachments/Attachment.js:27]: Do we need to add it again?

Comment (CulturalProfessor) [reply src/components/Attachments/Attachment.js:27]: No.

Comment (CulturalProfessor) [reply src/components/Attachments/Attachment.js:27]: Hey btw I was working on input-toolbar visibilty, is this good to have?

[Screencast from 18-02-23 04:53:40 PM IST.webm](https://user-images.githubusercontent.com/92238941/219860087-a8239444-588e-4802-b24f-0a8a55572d3f.webm)

**Pull request closed without merging**